<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Building Exemplars</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Task Setting</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Task</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>143</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Url</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_url">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_task_id">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>185</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Task ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_task">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="but_connect">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_file">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="but_open">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>45</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="WorkingArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>751</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Working Area</string>
   </property>
   <widget class="QTextEdit" name="input_subtask">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SubTask</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>153</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Subtask List</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Task List in Exemplars</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_task">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>351</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="list_task_content">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_actions">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="but_getstate">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get State</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_state">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Initial State</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_clear">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_append">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send to Subtask List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_delete">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_finalize">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_remove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_export">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_testjs">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test JS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
